ruby-hacktoberfest-2023/Rock_Paper_Scissor.rb

```diff
@@ -0,0 +1,53 @@
+r = rand (1..3)
+
+puts ("Choose rock paper or scissors...")
+user = gets.chomp.downcase
```

Comment:
I think, we need to add a check after getting the input from the user to see whether it's in the valid options(rock, paper, scissors), if not we need to show a message to the user, you choose an invalid option.